Pages/WishlistManagement/wishlist.html

```diff
@@ -0,0 +1,170 @@
+                <input type="search" placeholder="Search your choice" class="basic-textfield-outlined nav-search" id="search">
+                <button class="btn btn-md icon-md search-btn">
+                    <i class="fa fa-search " aria-hidden="true"></i>
+                </button>
+            </div>
+            <div class="nav-right">
+                <div class="header-icons">
+                    <div class="badge-icon badge-wrapper">
+                        <button class="btn btn-lg icon-lg">
+                            <a href="/Pages/WishlistManagement/wishlist.html" class="btn-link text-span hover-red">
+                                <i class="fa fa-heart"></i>
+                            </a>
+                        </button>
+                        <span class="badge-number secondary-bgcolor" id="wishlist-items">0</span>
+                    </div>
+                    <div class="badge-icon badge-wrapper">
+                        <button class="btn btn-lg icon-lg">
+                            <a href="/Pages/CartManagement/cart.html" class="btn-link text-primary hover-primary">
+                                <i class="fa fa-shopping-cart"></i>
+                            </a>
+                        </button>
+                        <span class="badge-number secondary-bgcolor" id="cart-items">0</span>
+                    </div>
+                    <button class="btn btn-lg icon-lg btn-user">
+                        <a href="" class="btn-link text-primary hover-primary">
+                            <i class="fa fa-user-circle-o"></i>
+                        </a>
+                        <div class="dropdown-content">
+                            <a href="Pages/Authentication/login.html" class="btn-link">Login</a>
+                            <a href="" class="btn-link">User Profile</a>
+                            <a href="" class="btn-link">Logout</a>
+                        </div>
+                    </button>
+                </div>
+                <button class="btn btn-md nav-right-btn">
+                    <a href="/Pages/Authentication/login.html" class="btn btn-link btn-primary-contained">Login</a>
+                </button>
+            </div>
+        </nav> 
+        <div class="background dis-grid">
+            <div class="product centered product-gryffindor product-houses">
+                <img src="/Assets/Gryffindor.png" class="img-responsive">
+            </div>
+            <div class="product centered product-ravenclaw product-houses">
+                <img src="/Assets/Ravenclaw.png" class="img-responsive">
+            </div>
+            <div class="product centered product-hufflepuff product-houses">
+                <img src="/Assets/Hufflepuff.png" class="img-responsive">
+            </div>
+            <div class="product centered product-slytherin product-houses">
+                <img src="/Assets/Slytherin.png" class="img-responsive">
+            </div>
```

Comment:
Whenever using images give an alt in case the image could not load on the user's end.

Reply:
Okay sure will change this